erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -7,9 +7,11 @@
 
 namespace erizo {
 
+const int ExternalOutput::PLI_INTERVAL = 5000;
```

Comment:
Okay.  So @j4y , does 5000 mean 5 seconds (ie: the minimum interval for packets)?